src/completers/git.zsh

```diff
@@ -58,24 +58,26 @@ PREVIEW_OPTIONS
 
 _fzf_complete_git() {
     setopt local_options extended_glob
-    local arguments=$@
+    local arguments=("${(Q)${(z)@}[@]}")
     local resolved_commands=()
 
     while true; do
-        local resolved=$(_fzf_complete_git_resolve_alias ${(Q)${(z)arguments}})
+        local resolved=$(_fzf_complete_git_resolve_alias $arguments)
         if [[ -z $resolved ]]; then
             break
         fi
 
+        # Ignores empty strings
```

Comment:
Write an inline comment in imperative mood. When describing a function, it should be in third-person singular instead.
```suggestion
        # Ignore empty strings
```